gallery/src/components/NxNexusPageHeader/NexusPageHeaderExample.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {faSave, faCog } from '@fortawesome/free-solid-svg-icons';
+
+import { NxButton, NxNexusPageHeader, NxFontAwesomeIcon } from '@sonatype/react-shared-components';
+
+const NexusPageHeaderExample = () => {
+  const links = [{
+    name: 'NxButton',
+    href: '#/pages/NxButton'
+  }, {
+    name: 'NxAlert',
+    href: '#/pages/NxAlert',
+    current: true
+  }];
+
+  function onButtonClick() {
+    alert('Click');
+  }
+
+  return (
+    <NxNexusPageHeader productInfo={{ name: 'Test Product', version: 'Fancy code name 1.0.0' }}
```

Comment:
"Fancy code name" here is your stand-in for the license type?  I'm wondering if that should be a separate field in the `productInfo` object.  If not, it should be clearly documented what all is intended to be included in the version string.

Reply:
> "Fancy code name" here is your stand-in for the license type? I'm wondering if that should be a separate field in the `productInfo` object. If not, it should be clearly documented what all is intended to be included in the version string.

My thinking was just to demonstrate that it can contain a string that includes text and a version number. In IQ that's the name of the license, but it might be something else in another product, or just the version number.

Reply:
> My thinking was just to demonstrate that it can contain a string that includes text and a version number. In IQ that's the name of the license, but it might be something else in another product, or just the version number.

Is that the intent from design or is the intent that this component should be more strict about it?

Reply:
> > My thinking was just to demonstrate that it can contain a string that includes text and a version number. In IQ that's the name of the license, but it might be something else in another product, or just the version number.
> 
> Is that the intent from design or is the intent that this component should be more strict about it?

Neither. To me this is just a string where we think people will put version info and in the case of IQ license + version info. I don't think that we can anticipate all of the potential uses, some might have text with no version number (e.g.: alpha, beta, pre-release), or just a number. To me we are responsible just that the formatting matches the design, but the designers are responsible for what will appear here.

Reply:
> Neither. To me this is just a string where we think people will put version info and in the case of IQ license + version info. I don't think that we can anticipate all of the potential uses, some might have text with no version number (e.g.: alpha, beta, pre-release), or just a number. To me we are responsible just that the formatting matches the design, but the designers are responsible for what will appear here.

My worry is that these are logically two different fields and while this design places them next to each other with equivalent styling, allowing us to combine them at the moment, a future design might draw a visual distinction between them.

Reply:
> > Neither. To me this is just a string where we think people will put version info and in the case of IQ license + version info. I don't think that we can anticipate all of the potential uses, some might have text with no version number (e.g.: alpha, beta, pre-release), or just a number. To me we are responsible just that the formatting matches the design, but the designers are responsible for what will appear here.
> 
> My worry is that these are logically two different fields and while this design places them next to each other with equivalent styling, allowing us to combine them at the moment, a future design might draw a visual distinction between them.

As discussed I have created a separate `prop` for the text called `meta`: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/a7a49bff24ed198dde1158a5573cddbdafdb923d